system-tests/README.md

```diff
@@ -1,122 +1,260 @@
 
-  for windows powershell
+# Register Consumer-US
+java -jar $Env:REGISTRATION_SERVICE_CLI_JAR_PATH -s="http://localhost:8184/api" participants add --request="{ \`"name\`": \`"consumer-us\`", \`"supportedProtocols\`": [ \`"ids-multipart\`" ], \`"url\`": \`"http://consumer-us:8282\`" }"
+```
 
-    ```powershell
-    $Env:REGISTRATION_SERVICE_CLI_JAR_PATH = "registration service client jar path"
-    # Execute command by copying it from shell script ./system-tests/resources/register-participants.sh or use git-bash to execute this shell script.
-    ```
+Set the environment variable `TEST_ENVIRONMENT` to `local` to enable local blob transfer test and then run `MVD` system test using the following command:
 
-- Run MVD system tests, and for that environment variable `TEST_ENVIRONMENT` must be set to `local` to enable local blob transfer test.
+```bash
+export TEST_ENVIRONMENT=local
+./gradlew :system-tests:test
+```
+
+_Note for Windows PowerShell, the following commands should be used:_
+
+```powershell
+$Env:TEST_ENVIRONMENT = "local"
+./gradlew :system-tests:test
+```
 
-    ```bash
-    export TEST_ENVIRONMENT=local
-    ./gradlew :system-tests:test
-    ```
+> [Storage Explorer](https://azure.microsoft.com/features/storage-explorer/) can be used to connect to the `Azurite` storage container on `127.0.0.1:10000` port and under `consumereuassets`, account transferred blob can be viewed.
 
-  for windows powershell
+### Local Test Resources
 
-    ```powershell
-    $Env:TEST_ENVIRONMENT = "local"
-    ./gradlew :system-tests:test
-    ```
+The following test resources are provided in order to run `MVD` locally. `system-tests/docker-compose.yml` uses it to start `MVD`.
 
-- [Storage Explorer](https://azure.microsoft.com/features/storage-explorer/) can be used to connect to Azurite storage container on `127.0.0.1:10000` port and under `consumereuassets` account transferred blob can be viewed.
+<br>
 
-#### Local test resources
+---
 
-Following test resources are provided in order to run MVD locally.`system-tests/docker-compose.yml` usages it to start MVD.
+<br>
+
+Each `EDC Connector` has its own set of Private and Public keys with java keystore e.g. `system-tests/resources/provider`. These were generated using the following commands:
+
+```bash
+# generate a private key
+openssl ecparam -name prime256v1 -genkey -noout -out private-key.pem
+# generate corresponding public key
+openssl ec -in private-key.pem -pubout -out public-key.pem
+# create a self-signed certificate
+openssl req -new -x509 -key private-key.pem -out cert.pem -days 360
+```
 
-- Each EDC Connector has its own set of Private and Public keys with java keystore e.g. `system-tests/resources/provider`.
+Generated keys are imported to keystores e.g. `system-tests/resources/provider/provider-keystore.jks`. Each keystore has password `test123`.
```

Comment:
path includes a `vault` component, e.g. `system-tests/resources/vault/provider/` (in multiple places)